107/src/main/java/org/ehcache/jsr107/Eh107MutableConfiguration.java

```diff
@@ -36,4 +52,20 @@ public Eh107MutableConfiguration(CompleteConfiguration<K, V> configuration) {
   public boolean isExpiryPolicyOverridden() {
     return this.expiryPolicyOverridden;
   }
+  
```

Comment:
What's the point of overriding equals and hashcode just to call the super methods?